graphql/http_handler.go

```diff
@@ -57,12 +58,17 @@ func Handler(apiURL string) func(w http.ResponseWriter, r *http.Request) {
 			queryPath = fieldCtx.Path().String()
 			args = fieldCtx.Args
 		}
-		grip.Error(message.WrapError(err, message.Fields{
+		errorMessage := message.WrapError(err, message.Fields{
 			"path":    "/graphql/query",
 			"query":   queryPath,
 			"args":    args,
 			"request": gimlet.GetRequestID(ctx),
-		}))
+		})
+		if strings.HasSuffix(err.Error(), ": context canceled") {
+			grip.Warning(errorMessage)
```

Comment:
If we're really only logging for debugging purposes, grip.Debug seems more correct.